lib/graphql_devise/mutations/send_password_reset.rb

```diff
@@ -18,7 +20,7 @@ def resolve(email:, redirect_url:)
           )
 
           if resource.errors.empty?
-            { authenticatable: resource }
+            { message: I18n.t('devise.passwords.send_instructions') }
```

Comment:
Please add the message to our locales file, that was done for all other messages. That way we won't be affected if d ecise changes that message or removes it